packages/roosterjs-editor-core/lib/editor/Editor.ts

```diff
@@ -359,9 +383,33 @@ export default class Editor {
             content = extractContentEvent.content;
         }
 
+        if (this.core.inDarkMode && !getInDarkMode) {
+            content = this.getLightModeContent(content);

```

Comment:
This makes it seem like otherwise you return the dark mode content (even if not in DarkMode). Maybe change name to something like `convertToLightModeContent`?

Reply:
`getLightModeContent` is more accurate, since this does not mutate the state of the editor on the dom

Reply:
gonna change it to `getColorNormalizedContent`